codalab/model/bundle_model.py

```diff
@@ -519,6 +521,32 @@ def make_condition(key, field, value):
                     clause = cl_bundle.c.uuid.in_(
                         alias(select([cl_worksheet_item.c.bundle_uuid]).where(condition))
                     )
+            elif key == '.before':
+                target_datetime = parser.isoparse(value)
```

Comment:
We don't have to use iso parse (we could just use parse). This is a stricter format, but I think it makes documentation and expectations simpler.